src/Model/Queue.php

```diff
@@ -32,7 +32,7 @@ public static function updateTime($id)
 
 		// Calculate the delay until the next trial
 		$delay = (($retrial + 3) ** 4) + (rand(1, 30) * ($retrial + 1));
-		$next = DateTimeFormat::utc(date('c', time() + $delay));
+		$next = DateTimeFormat::utc('now ' . $delay . ' seconds');
```

Comment:
You forgot the `+` in the string format.

Reply:
Oops ...